### ISSUE TITLE
Some confusions about extract my own pre-computed video features

### ISSUE THREAD

**Issue by dongfengxijian** [doc-doc/NExT-QA#22]
Thank you for excellent work！I want to extract my own NExT-QA features. As the data preparation section of redeame, I download the the highlighted NExTVideo. When I unzip it，I don't know how to divide the raw video into train set, valid set and test set. The VidOR website also didn't provide the test set.

**Comment by HU-xiaobai** [comment]
hello, could you see all the videos are valid? I also download the raw videos but some of them are not valid, which means I could not open the video.

**Comment by HU-xiaobai** [comment]
@dongfengxijian

**Comment by dongfengxijian** [comment]
> hello, could you see all the videos are valid? I also download the raw videos but some of them are not valid, which means I could not open the video.

I met the same problem. But when I extract frame with the scikit-video, it did‘t report error！Maybe the player is the reason,

**Comment by HU-xiaobai** [comment]
@dongfengxijian Hello, sorry to bother you again, could I ask if you could abstract the same appearance feature that the author provided? For example, for the validation set, I set the clip as 16 and I abstract the 8th or 9th(actually I compare all the frames in one clip) frame appearance vector of the first clip and compare the first frame vector of the provide appearance vector, both dimension are (2048,), but I find the appearance vector is different. how about yours?

**Comment by HU-xiaobai** [comment]
I now might could answer you question at the beginning. You could separate the video directly from the train.csv val.csv test.csv to make the video split

**Comment by dongfengxijian** [comment]
> @dongfengxijian Hello, sorry to bother you again, could I ask if you could abstract the same appearance feature that the author provided? For example, for the validation set, I set the clip as 16 and I abstract the 8th or 9th(actually I compare all the frames in one clip) frame appearance vector of the first clip and compare the first frame vector of the provide appearance vector, both dimension are (2048,), but I find the appearance vector is different. how about yours?

Should the appearance vector be same? I don't quite grasp your question

**Comment by dongfengxijian** [comment]
> I now might could answer you question at the beginning. You could separate the video directly from the train.csv val.csv test.csv to make the video split

Thank your for your guidance！

**Comment by HU-xiaobai** [comment]
@dongfengxijian , thanks for your answer! My question is: the appearance vector that I abstract is different from what the author provided, and how about you? Because we follow the same method, the appearance vector should be the same I think? because the weight of the resnext is fixed.

**Comment by dongfengxijian** [comment]
> @dongfengxijian , thanks for your answer! My question is: the appearance vector that I abstract is different from what the author provided, and how about you? Because we follow the same method, the appearance vector should be the same I think? because the weight of the resnext is fixed.

I didn't use the original model, so I can't answer your question. I‘m sorry about that.

**Comment by ghost** [comment]
> hello, could you see all the videos are valid? I also download the raw videos but some of them are not valid, which means I could not open the video.

I have the same issue
